app/views/admin/induction_periods/new.html.erb

```diff
@@ -33,7 +33,7 @@
           width: 4,
           inputmode: "numeric" %>
 
-      <%= f.govuk_collection_radio_buttons :induction_programme, training_programme_choices, :identifier, :name, legend: { text: 'Induction programme' }  %>
+      <%= f.govuk_collection_radio_buttons :induction_programme, induction_programme_choices, :identifier, :name, legend: { text: 'Induction programme' }  %>
```

Comment:
As far as I can tell, this wasn't intentional 🤔

The request spec used `fip` so never failed, but there was never an associated feature spec.

cc @peterdavidhamilton

Reply:
Good catch. On it.

Reply:
https://github.com/DFE-Digital/register-early-career-teachers-public/pull/946